### PR TITLE
Limit modules deployed to cucumber-jparest and jparest.

### PR DESCRIPTION
**Pull request by JWeston-HO**
This PR stops Demo and the ParentPom being deployed to packages.

### CODE REVIEW

File: cucumber-jparest/pom.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
     <properties>
       <spring.version>6.0.4</spring.version>
+      <maven.deploy.skip>false</maven.deploy.skip>
```
Comment (stephenkaye-ho) [inline cucumber-jparest/pom.xml:16]: Wed don't want to skip this do we

**Review by stephenkaye-ho — CHANGES_REQUESTED** [review]

File: cucumber-jparest/pom.xml (hunk 1 above)
Comment (JWeston-HO) [reply cucumber-jparest/pom.xml:16]: I have set the parent pom to be skip true.  Which means parent and all sub modules would be skipped in the deploy phase. Unless set too false. Meaning we won't skip them.

**Review by taomoh — APPROVED** [review]

File: demo/pom.xml [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
 	<properties>
 		<sonar.skip>true</sonar.skip>
+        <maven.deploy.skip>true</maven.deploy.skip>
```
Comment (taomoh) [inline demo/pom.xml:13]: Indent properly

Comment (JWeston-HO) [reply demo/pom.xml]: Fixed

**Review by taomoh — APPROVED** [review]

**Review by sewerynmi — APPROVED** [review]

**Review by stephenkaye-ho — APPROVED** [review]

**Pull request merged**